atef/tests/test_widgets.py

```diff
@@ -161,21 +168,21 @@ def test_serialization(group):
     print(yaml.dump(serialized))
 
 
-@parametrized_groups
-def test_create_widget(request: pytest.FixtureRequest, group: ProcedureStep):
-    widget = procedure_step_to_widget(group)
-    widget.show()
-    qt_utils.save_widget_screenshot(widget, prefix=request.node.name)
-    widget.close()
-    widget.deleteLater()
+# @parametrized_groups
+# def test_create_widget(request: pytest.FixtureRequest, group: ProcedureStep):
+#     widget = procedure_step_to_widget(group)
+#     widget.show()
+#     qt_utils.save_widget_screenshot(widget, prefix=request.node.name)
+#     widget.close()
+#     widget.deleteLater()
```

Comment:
Maybe this should include a short comment about why the test is disabled via comment